src/App.jsx

```diff
@@ -24,22 +25,17 @@ export const alex = {
 };
 
 export const App = () => (
-  <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
+  <>
+    <Person
+      person={misha}
+    />
```

Comment:
```suggestion
    <Person person={misha} />
```
we don't need to write props from new line if there is only one check all cases